### PR TITLE
cortex skills generate command added

### PR DESCRIPTION
**Pull request by ljha-CS**
# Checklist:
Please check you fulfill ALL of the relevant checkboxes
- [ ] Notified docs of any potential USER-facing changes
- [ ] Added short description of the change - with relevant motivation and context. 
- [ ] Branch has the ticket number in its name (along with a ticket summary)
- [ ] Commented the code, particularly in hard-to-understand areas
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Ran `npm test` and it passes
- [ ] Changes generate no new warnings

### CODE REVIEW

**Comment by jgielstra-cs** [comment]
Do we need to keep generator cortex? Should we embed the template in cortex-cli to avoid this dep??

**Comment by ljha-CS** [comment]
> Do we need to keep generator cortex? Should we embed the template in cortex-cli to avoid this dep??

Added templates in cli repo

File: src/commands/assets/templates/skill/rest-api/skill.yaml.hbs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    description: (optional)
+    defaultValue: POST
+    type: String
+inputs:
+  - name: input
+    title: input
+    routing:
+      all:
+        output: output
+        runtime: cortex/http
```
Comment (mburpo-CS) [inline src/commands/assets/templates/skill/rest-api/skill.yaml.hbs:35]: should be `cortex/external-api`
Also need to change type option in  generate from `REST API` to `External API`

Comment (ljha-CS) [reply src/commands/assets/templates/skill/rest-api/skill.yaml.hbs]: updated

**Pull request merged**